### ISSUE TITLE
let include use ../

### ISSUE THREAD

**Issue by jeremyBass** [mozilla/nunjucks#326]
in my case it would be very handy to be able to use `{% include "../pages/"+current_build+".html" -%}`

If I put the `pages` folder in the template folder and do `{% include "pages/"+current_build+".html" -%}` it works fine, but in this case the `pages` folder is context and shouldn't be in the template folder.  It would be nice if we could traverse the folders so I could keep my content and templates separate.

**Comment by SamyPesse** [comment]
:+1:

**Comment by devoidfury** [comment]
You can use multiple template folders by manually creating the loader instead of the automatic one.

```js
var cascade_loader = new nunjucks.FileSystemLoader(['./templates', './content']);
var env = new nunjucks.Environment(cascade_loader);
// ... hook env into your app or use it directly
```

**Comment by popomore** [comment]
Agreed, it should lookup always inside the directory. Relative path is useful, and throw exception when out of the box.

**Comment by SamyPesse** [comment]
@popomore :+1:

**Comment by jlongster** [comment]
fixed in #349, it's a pretty simple fix actually and you can't reach "outside the box"
